### DISCUSSION TITLE
How do I actually get access to MIMIC III csv files after being approved?

### DISCUSSION THREAD

**Discussion by yalinisen** [MIT-LCP/mimic-code#1487] [MIMIC-General]
Hi,
I was approved for access to MIMIC III (we are using that as it is the only resource that actually has progress notes).  However there is no indication on any of the pages as far as I can see on how to actually get access to csv files for local download (required for our project).  The links on web pages just seem to lead in a circle.  Please help.
Thank you,

**Comment by tompollard** [comment]
If you search for MIMIC-III on Google/PhysioNet, you should find the MIMIC-III project at:
https://physionet.org/content/mimiciii/

Here you will see a section named "Files" which contains options for accessing the data (either as CSV download, or on the cloud):
https://physionet.org/content/mimiciii/1.4/#files

**Comment by yalinisen** [comment]
Hi

Please see attached - does this mean I have to wait until my training is reviewed and approved? I passed the CITI.



Thank you,

Yalini






---- On Thu, 16 Feb 2023 16:08:00 -0500 Tom Pollard ***@***.***> wrote ---





If you search for MIMIC-III on Google/PhysioNet, you should find the MIMIC-III project at:
 https://physionet.org/content/mimiciii/
Here you will see a section named "Files" which contains options for accessing the data (either as CSV download, or on the cloud):
 https://physionet.org/content/mimiciii/1.4/#files

**Comment by tompollard** [comment]
If you check your training status at https://physionet.org/settings/training/, you'll see that your training is still under review. As MIMIC requires both CITI training and credentialing, you will not be able to access the data until your training is approved.

**Reply by tompollard** [reply]
(Side note, but you should have received emails to explain this. If you did not, please check your spam folder!)

**Comment by ndmc41030018-cyber** [comment]
How long will it take to review the training records?
